src/group_theory/free_abelian_group_finsupp.lean

```diff
@@ -0,0 +1,145 @@
+/-
+Copyright (c) 2021 Johan Commelin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johan Commelin
+-/
+
+import group_theory.free_abelian_group
+import data.finsupp.basic
+
+/-!
+# Isomorphism between `free_abelian_group X` and `X →₀ ℤ`
+
+In this file we construct the canonical isomorphism between `free_abelian_group X` and `X →₀ ℤ`.
+We use this to transport the notion of `support` from `finsupp` to `free_abelian_group`.
+
+## Main declarations
```

Comment:
My apologies: I committed the change from `declerations` to `declarations` rather than `adding` a comment.

Reply:
No worries!